molecule/default/tests/test_dashboard.py

```diff
@@ -22,9 +22,16 @@ def test_find_openrc_on_opt(host):
 def test_find_openrc_on_etc(host):
     assert host.file("/etc/openrc").exists
 
+
 # Experimental test
-def test_find_cinder_client(host):
+def test_cinder_service(host):
```

Comment:
The test is only running when there is '/root/openrc' and 'cinder client' in the host